### DISCUSSION TITLE
Will MonitorControl have smaller steps in in/decrease screen brightness on Apple Studio Monitor (with M4 Mini Mac)

### DISCUSSION THREAD

**Discussion by cglennx** [MonitorControl/MonitorControl#1678] [Q&A] [answered]
I just got my new M4 Mini Mac and the Apple Studio Monitor.  I have autism and am sensitive to light.  At night, the lowest brightness of the monitor is just a little too dark, but the first step up is a little too bright.  Will this software have smaller steps in increasing and decreasing brightness?

**Comment by cglennx** [comment]
> I just got my new M4 Mini Mac and the Apple Studio Monitor. I have autism and am sensitive to light. At night, the lowest brightness of the monitor is just a little too dark, but the first step up is a little too bright. Will this software have smaller steps in increasing and decreasing brightness?

Is anyone out there?  I don't know if this question is stupid or just no one has an answer.

**Reply by waydabber** [reply]
Yes, it should allow further dimming using software control.

**Reply by cglennx** [reply]
Thank you for the reply.  But will it adjust the steps of the dimming?  I am using the Apple Studio Monitor.  At night, the lowest setting is dark enough, actually just right.  But some times I need it just a little brighter.  The issue is that one step up in brightness is too much.  I don't know if this is hard-wired in the monitor, of if software can give me more control, that is, maybe a half-step up in brightness for each press of fn+F1/F2 on the keyboard.

**Reply by waydabber** [reply, marked as answer]
It can switch to make fine OSD scale default instead of the full OSD step.

**Reply by cglennx** [reply]
Thank you!  If it works, how do I make a modest financial contribution to you?

**Reply by waydabber** [reply]
Oh you are welcome. If you'd like to support the open source project, simply donate any amount here:

https://opencollective.com/monitorcontrol

Every bit counts. :)
